src/main/resources/reference.conf

```diff
@@ -2,7 +2,6 @@
 # Google Cloud dataproc configuration
 dataproc {
   applicationName = "firecloud:leonardo"
-  dataprocDockerImage = "gcr.io/broad-dsde-prod/leonardo-jupyter:prod"
```

Comment:
this is removed because it's duplicated by leonardo.conf

Reply:
Yeah.. makes sense.